src/main/java/io/codiga/plugins/jetbrains/assistant/transformers/VariableIndentation.java

```diff
@@ -0,0 +1,60 @@
+package io.codiga.plugins.jetbrains.assistant.transformers;
+
+import com.intellij.application.options.CodeStyle;
+import com.intellij.openapi.actionSystem.LangDataKeys;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiFileFactory;
+import io.codiga.plugins.jetbrains.model.CodingAssistantContext;
+
+public class VariableIndentation implements VariableTransformer {
+
+  /**
+   * Set file's correct indentation in the recipe
```

Comment:
Put a more descriptive comment. Especially since we look exactly at how the editor uses tab vs space and we adapt the recipe with the `&[CODIGA_INDENTATION]` variable.